### PR TITLE
Support IAM role for RDS check, and rubocop fixes

### PR DESCRIPTION
**Pull request by johntdyer**
## Pull Request Checklist

**Is this in reference to an existing issue?**

#### General

- [x] Update Changelog following the conventions laid out on [Keep A Changelog](http://keepachangelog.com/)

- [x] Update README with any necessary configuration snippets

- [x] RuboCop passes

- [x] Existing tests pass 

#### Purpose

Add support for IAM auth for rds check 

#### Known Compatablity Issues

n/a

### CODE REVIEW

**Comment by johntdyer** [comment]
Any idea why travis didnt run on this PR?

**Comment by sstarcher** [comment]
no idea try rebasing it

File: bin/check-rds.rb [hunk 1]
```diff
@@ -102,6 +102,25 @@ class CheckRDS < Sensu::Plugin::Check::CLI
          description: 'Continue if CloudWatch provides no metrics for the time period',
          default: false
 
+  option :use_iam_role,
```
Comment (sstarcher) [inline bin/check-rds.rb]: if credentials are not supplied the sdk will default to use the iam instance profile.

**Comment by sstarcher** [comment]
aws sdk will use the iam instance profile if credentials are not provided

**Pull request closed without merging**